### PR TITLE
next step working with multilang injection

### PR DESCRIPTION
**Pull request by AlexTriada**

### CODE REVIEW

File: AntistasiOfficial.Altis/Missions/FND_ExpDealer.sqf [hunk 1]
```diff
@@ -45,7 +45,7 @@ while {!(_break) && (count _roads > 0)} do {
 	};
 };
 
-if !(_break) exitWith {[[petros,"globalChat","STR_TSK_TD_CHAT_1"],"commsMP"] call BIS_fnc_MP;};
```
Comment (AlexTriada) [inline AntistasiOfficial.Altis/Missions/FND_ExpDealer.sqf]: if you wont to localize globalChat in commsMP use locGlobalChat

File: AntistasiOfficial.Altis/Missions/FND_ExpDealer.sqf [hunk 2]
```diff
@@ -139,7 +139,7 @@ if (random 8 < 1) then {
 
 waitUntil {sleep 1; (dateToNumber date > _fechalimnum) || !(alive Devin) || ((Devin distance _posCmp) > 50) || ({(side _x isEqualTo side_blue) && (_x distance Devin < 200)} count allPlayers > 0)};
 
-{if (isPlayer _x) then {[petros,"hint","STR_TSK_TD_CHAT_2"] remoteExec ["commsMP",_x]}} forEach ([200,0,Devin,"BLUFORSpawn"] call distanceUnits);
```
Comment (AlexTriada) [inline AntistasiOfficial.Altis/Missions/FND_ExpDealer.sqf]: if you wont to localize hint in commsMP use locHint

**Comment by StefArma** [comment]
Interesting!!

**Comment by zalexki** [comment]
don't understand is this ready to merge or WIP or more of an advice of how to ?

**Comment by AlexTriada** [comment]
WIP

**Comment by zalexki** [comment]
ok @AlexTriada you can reopen and i'll add WIP label if you prefer

**Pull request closed without merging**